### DISCUSSION TITLE
Unable to deploy my contract

### DISCUSSION THREAD

**Discussion by Ifra001** [Cyfrin/foundry-full-course-cu#2819] [Q&A]
I am trying to deploy my MerkleAirdrop contract and I wrote this in the terminal
```solidity
forge script script/DeployMerkle.s.sol:DeployMerkle --rpc-url $SEPOLIA_RPC_URL --private-key $PRIVATE_KEY --broadcast 
```
But my transaction keeps on failing
```solidity
Error: Failed to send transaction

Context:
- server returned an error response: error code -32000: future transaction tries to replace pending
```

What do I do? I have no transactions pending on my account so then this should go through. Right?

**Comment by EngrPips** [comment]
Hello @DontMind-me, I haven't run into this issue before, so it is hard to tell what is wrong. I think you should check the account associated with the `private key` you are using for the deployment process and ensure there is truly no pending transaction.

**Reply by Ifra001** [reply]
I'll do that! but I was also suspecting that I needed to write something more after the ```--broadcast``` but I forgot exactly. I this it was the ```etherscan verfication```  or something like that but I believe it is optional

**Reply by EngrPips** [reply]
Verification is optional I believe.
